README.md

```diff
@@ -1,14 +1,35 @@
 # D Project Idea repository
 
 This repository hosts and organises a list of impactful projects in the D ecosystem.
-Milestones are used to group project ideas and keep track of them.
 The previously used vision document is superseeded by the continous, changing vision build up from the milestones of this repo.
 
-## [Project tracker](https://github.com/dlang/projects/issues)
+## [Browse project ideas](https://github.com/dlang/projects/issues)
 
-## Label Types
-Different labels are used to help organise the project ideas.
-Please inspect the descriptions of the labels to understand there exact meaning.
+Note: some of these projects are not necessarily endorsed by the D Language Foundation. Endorsed projects are marked with the "vision" label.
 
-## University theses
-If you are a student and are interesting in working on a project idea for your thesis, please do not hesitate to leave a comment on the issue and/or conntact people mentioned with that issue.
+Everyone is welcome to submit new project ideas. However, please keep in mind that this repository tries to collect high-level goals and project. A good rule of thumb is that if a project can't be split into smaller parts, it might belong to its respective issue tracker.
+
+Bad Example: Fix Issue 12345 in DMD
+
+Good Example: Improve DMD user experience on Windows (includes a list of issues)
+
+## Voting
+
+As projects are suggested by individual people, it's hard to judge how important they are to community. Hence, please vote on them via GitHub reactions as this helps us a lot in priotizing issues.
+
+## Vision projects
+
+Projects that are marked with the "vision" labelled are vision projects of the D Language Foundation.
+These projects are seen as key projects to D's success by the leadership.
+
+## Milestones
+
+[Milestones](https://github.com/dlang/projects/milestones) are used to group project ideas and keep track of them.
+
+### University thesis
+
+If you are a student and are interesting in working on a project idea for your thesis, please do not hesitate to leave a comment on the issue and/or contact the persons mentioned in the respective issue.
```

Comment:
Should be 

If you are a student and are interest**ed** (copy pasta mistake). 

Sorry too jet lagged to do it myself.